### DISCUSSION TITLE
Allow any emoji when creating the spotlight (including different skin tones)

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#32] [Discussions] [answered]
Right now, there's only a chosen subset of emojis available in emoji picker when you create the spotlight. As an example, I can't use a different skin tone for the emojis (I also tried changing my preferred skin tone in user settings), but aside from that I also can't use emojis like 🍆, while I can use 🍇 or 🥚.

**Comment by becca** [comment, marked as answer]
hey @jack1142 thank you for your feedback. we're about to release another update to Spotlights/Pinned Discussions shortly where the only emojis visible in them will be that of the Category in which the Discussions belongs. We're adding other easy ways to customize those Spotlights among other aspects of Discussions.

**Comment by fadol83-lab** [comment]
iconic-post-464511-j3
